modules/docker/Makefile.clean

```diff
@@ -0,0 +1,39 @@
+## Cleanup docker volumes.             WARNING!!! IT WILL DELETE ALL UNUSED VOLUMES
+docker/clean/volumes:
+	@echo "Cleanup volumes"
+	@docker volume prune
+	@echo "Done"
```

Comment:
nit: IMO these seem a bit unnecessary. I think it's better that people have to learn to use the underlying tools like `docker` than have a bunch of Makefile commands that are essentially aliases for existing commands.

Even your `docker/clean` command is basically just an alias for `docker system prune`.

Just my two cents. Feel free to ignore this.

Reply:
That's fair. What do you think is a good heuristic for when to create a target vs. not? 

I'll take these out.

Reply:
Good question. How about these criteria?

If your Makefile target:
* implements a single command and doesn't use any non-default options
* has no dependencies on other targets
* is not something we need to standardize as part of CI, pre-commit checks, testing, etc.

then it probably doesn't need to be a Makefile target.

Reply:
The `docker/clean/images/all` command doesn't fit the above criteria.
```
@docker images -q --no-trunc | xargs -I{} docker rmi {}
```

It seems pretty destructive for a command without any confirmation prompt though. Personally I think it might be better to not have this as part of the build harness.

Reply:
Agreed, I will just remove this file with all the clean commands.